File: app/src/main/java/com/tbm/dogs/Helper/Shared.java
package com.tbm.dogs.Helper;

import android.content.Context;
import android.content.SharedPreferences;

public class Shared {
    SharedPreferences shared;
    Context context;

    public Shared(Context context) {
        this.context = context;
        shared = context.getSharedPreferences("dogs",Context.MODE_PRIVATE);
    }
    public void saveInfoUser(String s){
        shared.edit().putString("user",s).apply();
    }
    public String getInfoUser(){
        return shared.getString("user","");
    }
    public void saveTokenFCM(String token){ shared.edit().putString("tokenfcm",token).apply();}
    public String getTokenFCM() { return shared.getString("tokenfcm","");}
}

File: app/src/main/java/com/tbm/dogs/Helper/Shared.kt
package com.tbm.dogs.Helper

import android.content.Context
import android.content.SharedPreferences

class Shared(internal var context: Context) {
    internal var shared: SharedPreferences
    val infoShiper: String
        get() = shared.getString("shiper", "")
    val tokenFCM: String
        get() = shared.getString("tokenfcm", "")
    val user:String
        get() = shared.getString("user","")
    val pass:String
        get() = shared.getString("pass","")

    init {
        shared = context.getSharedPreferences("dogs", Context.MODE_PRIVATE)
    }

    fun saveInfoShiper(s: String) {
        shared.edit().putString("shiper", s).apply()
    }

    fun saveUserPass(user:String,pass:String){
        shared.edit().putString("user",user).apply()
        shared.edit().putString("pass",pass).apply()
    }

    fun saveTokenFCM(token: String) {
        shared.edit().putString("tokenfcm", token).apply()
    }
}
